src/components/Modal/Modal.module.css

```diff
@@ -91,10 +91,16 @@
 
   width: 22.5rem;
   --modal-horizontal-padding: var(--eds-size-4);
+}
+
+.modal__content--md {
```

Comment:
For some reason I found the styles confusing because the large style includes medium but that's not obvious from the css module, so I would like a comment  here on the medium style that it's used for both medium and large, or maybe rename the class to make that clearer.

Reply:
Mmm that makes sense, I'll leave a comment